spec/support/seeds/seeds_e2e.rb

```diff
@@ -101,6 +101,43 @@
 
 ##############################################################################
 
+seeder.create_if_doesnt_exist(User, "email", "punctuated-name-user@forem.local") do
+  user = User.create!(
+    name: "User \"The test breaker\" A'postrophe  \\:/",
```

Comment:
for local testing in development I modified the admin username to be Mc'Admin (I was going to make it O'Admin but stopped)

![edited-username](https://user-images.githubusercontent.com/1237369/156421705-ac962442-e189-41f8-bb66-901003b48c17.png)


On main I see the json parse error (oddly, the follow buttons for other users still worked, that's progress)

![error-on-main](https://user-images.githubusercontent.com/1237369/156421780-3682e4b2-0dae-4fe5-add2-26fea7f0ea80.png)


This error goes away in the PR branch